File: Assignment_No_11/program4.java
import java.util.*;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }
        
        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print("# "+iCnt+" * ");
        }
    }
}

class program4
{
    public static void main(String arr[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.print("Enter number of elements : ");
        iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}

File: Assignment_No_41/program4.cpp
#include<iostream>
using namespace std;

void Display(int iNo)
{
    static char ch = 'A';
    static char i = 1;

    if (i <= iNo)
    {
        printf("%c ",ch);
        ch++;
        i++;

        Display(iNo);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}